Reply/app/src/main/res/layout/fragment_email.xml

```diff
@@ -16,128 +16,167 @@
+          <Button
+              android:id="@+id/email_reply_all_button"
+              android:layout_width="0dp"
+              android:layout_height="wrap_content"
+              android:layout_marginTop="@dimen/grid_1"
+              android:layout_marginStart="@dimen/grid_1"
+              android:text="@string/email_reply_all"
+              app:layout_constraintEnd_toEndOf="parent"
+              app:layout_constraintStart_toStartOf="@+id/guidelineLeft"
+              app:layout_constraintTop_toBottomOf="@+id/attachment_recycler_view" />
+          <ImageView
+              android:id="@+id/sender_profile_image_view"
+              android:layout_width="@dimen/email_sender_profile_image_size"
+              android:layout_height="@dimen/email_sender_profile_image_size"
+              android:contentDescription="@string/email_sender_profile_content_desc"
+              android:scaleType="centerCrop"
+              app:glideCircularCrop="@{true}"
+              app:glideSrc="@{email.sender.avatar}"
+              app:layout_constraintStart_toStartOf="parent"
+              app:layout_constraintTop_toTopOf="parent"
+              tools:src="@drawable/avatar_3" />
+          <ImageView
+              android:id="@+id/email_star_image_view"
+              android:layout_width="@dimen/email_star_image_size"
+              android:layout_height="@dimen/email_star_image_size"
+              android:padding="@dimen/grid_1"
+              android:background="?attr/selectableItemBackgroundBorderless"
+              android:contentDescription="@{email.starred ? @string/email_starred_content_desc : @string/email_unstarred_content_desc}"
+              android:foreground="@drawable/ic_circle"
+              android:src="@{email.starred ? @drawable/ic_star_filled : @drawable/ic_star_border}"
+              app:glideCircularCrop="@{true}"
+              app:layout_constraintEnd_toEndOf="parent"
+              app:layout_constraintTop_toTopOf="parent" />
+          <TextView
+              android:id="@+id/email_time_sent_text_view"
+              android:layout_width="0dp"
+              android:layout_height="wrap_content"
+              android:layout_marginTop="@dimen/grid_0_5"
+              android:layout_marginStart="@dimen/grid_1"
+              android:text="@{email.timeSent}"
+              android:textAppearance="?attr/textAppearanceBody2"
+              app:layout_constraintStart_toEndOf="@+id/sender_profile_image_view"
+              app:layout_constraintTop_toBottomOf="@+id/email_sender_text_view"
+              tools:text="2 hr ago" />
+          <!-- TODO: Temporary hardcoded black, remove when updating to M3 Typography  -->
+          <TextView
+              android:id="@+id/email_sender_text_view"
+              android:layout_width="0dp"
+              android:layout_height="wrap_content"
+              android:layout_marginStart="@dimen/grid_1"
```

Comment:
Make this `marginHoriontal`. If for some reason the sender text view is really long, we wouldn't want it to touch the end of the card